janitor/functions.py

```diff
@@ -144,9 +144,9 @@ def _remove_special(col):
 @pf.register_dataframe_method
 def remove_empty(df):
     """
-    Drop all rows and columns that are completely null.
+    Drop all rows and columns that are completely null. This method also resets the index(by default) since it doesn't make sense to preserve the index of a completely empty row.
```

Comment:
@dsouzadaniel thanks for putting this in! Could you also ensure that the docstring is 79 characters per line, max?